R/auto-test.R

```diff
@@ -103,9 +103,13 @@ auto_test_package <- function(pkg = ".", reporter = default_reporter(), hash = T
 
     tests <- changed[starts_with(changed, test_path)]
     helper <- tests[starts_with(basename(tests), "helper-")]
-    tests <- setdiff(tests, helper)
     code <- changed[starts_with(changed, code_path)]
 
+    # Remove helper from test and add it to code (if a helper changed,
+    # like for code, reload all and rerun all tests)
+    tests <- setdiff(tests, helper)
```

Comment:
Can you please move `helper <- ...` into this block as well? I think that makes it all hang together better

Reply:
Sure, just did it!